templates/types/streaming/nextjs/app/components/ui/chat/chat-input.tsx

```diff
@@ -63,17 +65,12 @@ export default function ChatInput(
     attachments: JSONValue[] | undefined,
   ) => {
     e.preventDefault();
-    props.append!(
-      {
-        content: props.input,
-        role: "user",
-        createdAt: new Date(),
-        annotations: attachments,
-      },
-      {
-        data: { imageUrl, csvFiles: files },
-      },
-    );
+    props.append!({
+      content: props.input,
+      role: "user",
+      createdAt: new Date(),
+      annotations: attachments,
+    });
     setImageUrl(null);
     resetUploadedFiles();
     props.setInput!("");
```

Comment:
is this not something we also want generally in onSubmit to happen (even if there's no attachment)?

Reply:
i mean the `props.setInput!("");` - should it not be in `onSubmit`?